examples/waspello/src/cards/Main.css

```diff
@@ -235,6 +235,10 @@ button.open-add-list {
   border-radius: 3px;
 }
 
+.list--dragging {
```

Comment:
I think we should add transitions to make this look smoother:

```css
.list {
  ...
  transition: transform 200ms ease-in-out;
}

.list--dragging {
  transform: rotate(2deg);
}

.list--drop-animating {
  transform: rotate(0deg);
}
```

I'd do the same for the `.list-card` component. 

Also, potential improvement in readability is to maybe declare CSS vars for the angle e.g. `--list-rotate-angle: 2deg` and then use it `var(--list-rotate-angle)`.

Reply:
I added suggested changes. I didn't extract the `2deg` to a separate variable as it is used only once (when dragging the whole list 4 degs are used), moving its definition to a different place could make the code harder to follow.

Reply:
P.S. I still need to test the whole list dragging, but it won't work until https://github.com/wasp-lang/wasp/pull/2414 is resolved and merged.